benches/tokenizer.rs

```diff
@@ -29,7 +29,7 @@ struct Sink;
 impl TokenSink for Sink {
     type Handle = ();
 
-    fn process_token(&mut self, token: Token) -> TokenSinkResult<()> {
+    fn process_token(&mut self, token: Token, line_number: u64) -> TokenSinkResult<()> {
```

Comment:
Let's use `_line_number` here and elsewhere to avoid warnings about unused variables in code that never makes use of it.